### DISCUSSION TITLE
Breakble Math Equation

### DISCUSSION THREAD

**Discussion by suzhiking** [typst/typst#5230] [Q&A]
Hi, I referred to this solution https://github.com/typst/typst/pull/4226 to make math equation breakable just like LaTex,  but 
```typst
#set math.equation(block: true)
#show math.equation: set block(breakable: true)

#lorem(600)
$ 
a &= b\
b &= c\
c &= d\
e &= f\
g &= h\
$
```
<img width="488" alt="Screenshot 2024-10-17 at 10 10 56 PM" src="https://github.com/user-attachments/assets/29535247-9f67-453c-adc5-887bc8697e7a">

still break whole math environment to new page. How can I resolve this?

**Comment by Enivex** [comment]
This will only work if you're using the 0.12 preview

**Reply by suzhiking** [reply]
Thanks! How can I set to use 0.12 preview? I'm using [typst-preview.nvim](https://github.com/chomosuke/typst-preview.nvim).
